sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1950,8 +1950,53 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("create datasource table with a non-existing location") {
+    withTable("t", "t1") {
+      withTempDir {
+        dir =>
+          dir.delete()
+          spark.sql(
+            s"""
+               |CREATE TABLE t(a int, b int)
+               |USING parquet
+               |LOCATION '$dir'
+             """.stripMargin)
```

Comment:
The test case is already pretty long. Please reduce the sql statement to a single line.